packages/pn-pa-webapp/src/pages/components/Notifications/types.ts

```diff
@@ -1,4 +1,4 @@
-import { ReactNode } from "react";
+import { ReactNode } from 'react';
```

Comment:
su develop questo file non esiste più, ma c'è un nuovo file dei types della tabella in pn-commons: `NotificationsTable.ts`.
Se la vostra intenzione è modificare quello non ci sono problemi (vedo anzi che l'avete utilizzato in seguito). Valutiamo anche di rinominare il nome del file